src/App/Fossa/ManualDeps.hs

```diff
@@ -187,12 +203,12 @@ toSourceUnit root depsFile manualDeps@ManualDependencies{..} maybeApiOpts vendor
   -- validated without endpoint interactions.
   rdeps <- case maybeApiOpts of
```

Comment:
I can't get this to let me give a suggestion for the following 5 lines, but I think this is cleaner
```
  rdeps <- case org of
    Just orgValid ->  
      traverse (`validateRemoteDep` orgValid) remoteDependencies
    Nothing -> pure remoteDependencies
```
Or you could do `maybe

Reply:
🤔 yeah that's fair!